cxf/src/main/java/org/opennms/core/wsman/cxf/CXFWSManClient.java

```diff
@@ -297,7 +303,9 @@ public Node get(String resourceUri, Map<String, String> selectors) {
         } catch (RuntimeException e) {
             throw wrapException(e);
         } finally {
-            destroy(transferer);
+            if (transferer != null) {
+                destroy(transferer);
```

Comment:
we can check for null in destroy method instead of checking it in multiple places